### ISSUE TITLE
Add Skeleton Loading Effect for Weather Data Fetching 

### ISSUE THREAD

**Issue by Gagandeep-2003** [Mohit5Upadhyay/WeatherApplication#44]
When fetching weather data, the application currently shows no indication of loading, leading to a poor user experience. Implement a skeleton loading effect for the current weather and 5-day forecast sections while data is being fetched. 

This will provide users with visual feedback during data retrieval, enhancing the overall experience and reducing the perceived waiting time.

**Comment by Mohit5Upadhyay** [comment]
Yes, @Gagandeep-2003 go ahead to implement the skeleton loading effect during data fetching that will overall enhance the UI/UX of the application

**Comment by Sawan-Kushwah** [comment]
can you assignee me i want to work on this ?
